test/integration/puppetlabs/services/certificate_authority/certificate_authority_int_test.clj

```diff
@@ -79,34 +83,41 @@
         app
         {:certificate-authority {:certificate-status
                                  {:client-whitelist ["notlocalhost"]}}
-         :authorization         {:rules [{:match-request
-                                                 {:path "/puppet-ca/v1/certificate"
-                                                  :type "path"}
-                                          :allow ["localhost"]}]}}
-        (testing "are denied"
-          (doseq [ca-mount-point ca-mount-points
-                  endpoint ["certificate_status/localhost"
-                            "certificate_statuses/ignored"]]
-            (testing (str "for the " endpoint " endpoint")
-              (let [response (http-get (str ca-mount-point endpoint))]
-                (is (= 403 (:status response))
-                    (ks/pprint-to-string response))))))
-        (testing "are allowed when allowed by rule to the certificate endpoint"
-          (doseq [ca-mount-point ca-mount-points]
-            (let [response (http-get (str ca-mount-point
-                                          "certificate/localhost"))]
-              (is (= 200 (:status response))
-                  (ks/pprint-to-string response)))))))))
+         :authorization         {:version 1
+                                 :rules [{:match-request
+                                          {:path "/puppet-ca/v1/certificate"
+                                           :type "path"}
+                                          :allow ["localhost"]
+                                          :sort-order 1
+                                          :name "cert"}]}}
+        (logutils/with-test-logging
```

Comment:
Same comment here on pushing the `with-test-logging` down to just being around the `http-get` for the `denied` test.